src/ui/AccountDropdown/index.svelte

```diff
@@ -67,23 +72,19 @@
       </section>
 
       <hr />
-    {:else}
-      <section>
+    {/if}
+
+    <section>
+      {#if !currentUser}
         <a
-          href="/login"
+          href="https://app.santiment.net/login"
```

Comment:
`/login` is universal page
```suggestion
          href="/login"
```